lang/cs/Org.Apache.REEF.Driver/Evaluator/IFailedEvaluator.cs

```diff
@@ -32,7 +33,7 @@ public interface IFailedEvaluator : IIdentifiable
     {
         EvaluatorException EvaluatorException { get; set; }
```

Comment:
@markusweimer That's actually a separate item (REEF-666) that I'm working on. If you want to incorporate both, let me know.